var/spack/repos/builtin/packages/charmpp/package.py

```diff
@@ -18,10 +18,12 @@ class Charmpp(Package):
     (your laptop) to the largest supercomputers."""
 
     homepage = "http://charmplusplus.org"
-    url      = "https://charm.cs.illinois.edu/distrib/charm-6.8.2.tar.gz"
+    url      = "http://charm.cs.illinois.edu/distrib/charm-6.8.2.tar.gz"
     git      = "https://github.com/UIUC-PPL/charm.git"
 
-    version("develop", branch="charm")
+    version("develop", branch="master")
+
+    version('6.10.1', sha256='ab96198105daabbb8c8bdf370f87b0523521ce502c656cb6cd5b89f69a2c70a8')
```

Comment:
Now that #15667 is merged, these arent' necessary anymore.

Reply:
This was causing a merge conflict and preventing the CI from running so I manually copied the contents of https://github.com/spack/spack/pull/15667 over here. I guess I could've added that commit on my branch but it was simple enough to cut/paste.

Reply:
I think this might cause conflicts when merging later on. I think it might be easier to just run `git merge develop` on your branch and push it.

Reply:
done!